pkgs/applications/window-managers/hyprwm/hyprland/plugins.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, gcc13Stdenv
+, hyprland
+}:
+
+let
+  mkHyprlandPlugin =
+    a@{ pluginName, ... }:
+    gcc13Stdenv.mkDerivation (a // {
+      pname = "${pluginName}";
+      nativeBuildInputs = [ pkg-config ]
+        ++ (a.nativeBuildInputs or [ ]);
+      buildInputs = [ hyprland ]
+        ++ hyprland.buildInputs
+        ++ (a.buildInputs or [ ]);
+      meta = a.meta // {
+        description = a.meta.description ++
+          ". Plugins can be installed via a plugin entry in the hyprland NixOS option or home-manager config.";
+      };
```

Comment:
There is no nixos option for this because it can only be specified inside the config. I'd also write it as `\n\n<text?`

Reply:
my bad I was convinced it was also existing. Corrected